README.md

```diff
@@ -41,24 +41,36 @@ We have introduced this feature with [APIM CLI](https://github.com/wso2/product-
 
 ##### Deploying APIM in  K8S Cluster
 
-Kubernetes artifacts to deploy APIM and APIM analytics deployment are shipped with the this distribution.
+Kubernetes artifacts to deploy APIM and APIM analytics deployment are shipped with the this distribution. Below commands allow users to run API Manager default profile and API Manager Analytics profile in kubernetes runtime. Please note you will need to have enough menory and CPU to run below commands.
 
 Navigate to wso2am-k8s-crds/apim-operator/apim-deployment/api-manager
 
-- Deploy API Manager in Kubernetes Cluster
+- Deploy API Manager in Kubernetes Cluster. Following command will deploy WSO2 API Manager all in one profile in kubernetes. With following command API Manager default profile will start run on kubernetes runtime under WSO2 namespace. Please see below command and its output. 
```

Comment:
Let's remove references to profiles. Many users coming here won't be aware of these.